app/app/urls.py

```diff
@@ -120,11 +121,18 @@
     url(r'^funding/details/?', dashboard.views.bounty_details, name='funding_details'),
 
     # Tips
-    url(r'^tip/receive/?', dashboard.views.receive_tip, name='receive_tip'),
-    url(r'^tip/send/2/?', dashboard.views.send_tip_2, name='send_tip_2'),
-    url(r'^tip/send/?', dashboard.views.send_tip, name='send_tip'),
-    url(r'^send/?', dashboard.views.send_tip, name='tip'),
-    url(r'^tip/?', dashboard.views.send_tip, name='tip'),
+    url(
+        r'^tip/receive/v2/(?P<pk>.*)/(?P<txid>.*)/(?P<network>.*)?',
+        dashboard.tip_views.receive_tip_v2,
+        name='receive_tip'
+    ),
+    url(r'^tip/receive/?', dashboard.tip_views.receive_tip_legacy, name='receive_tip_legacy'),
+    url(r'^tip/send/4/?', dashboard.tip_views.send_tip_4, name='send_tip_4'),
+    url(r'^tip/send/3/?', dashboard.tip_views.send_tip_3, name='send_tip_3'),
+    url(r'^tip/send/2/?', dashboard.tip_views.send_tip_2, name='send_tip_2'),
```

Comment:
@owocki @SaptakS  @mbeacom  should we rename this into something more meaning full ? ( we can do it in a follow up PR)
 `/2`, `/3` seems cryptic :P

Reply:
the numbers are the step numbers.. sequential numbers seem straightforward to me..